### DISCUSSION TITLE
World change - fly disable why?

### DISCUSSION THREAD

**Discussion by moda84** [EssentialsX/Essentials#4837] [Q&A] [answered]
Hi, if player change world fly disable why?
How I can enable save fly mode when player change world?
Config: 
https://pastebin.com/9pJwgM6F

**Comment by moda84** [comment]
Plugin ver.: EssentialsX-2.19.3
Server ver.: 
Current: git-Purpur-1579 (MC: 1.18.2)*
Previous: git-Purpur-1555 (MC: 1.18.1)

**Comment by Chew** [comment, marked as answer]
Duplicate of https://github.com/EssentialsX/Essentials/issues/4325
